noxfile.py

```diff
@@ -53,49 +53,49 @@ def parse_supported_python_versions() -> list[str]:
 
 @nox.session()
 def bandit(session: Session) -> None:
-    """Run bandit on cachi2 directory and noxfile.py."""
+    """Run bandit."""
     install_requirements(session)
-    cmd = "bandit -c pyproject.toml -r cachi2 noxfile.py"
+    cmd = "bandit -c pyproject.toml -r hermeto noxfile.py"
     session.run(*cmd.split(), *session.posargs, silent=True)
 
 
 @nox.session()
 def black(session: Session) -> None:
-    """Run black on cachi2 and tests directories and noxfile.py."""
+    """Run black."""
     install_requirements(session)
-    cmd = "black --check --diff cachi2 tests noxfile.py"
+    cmd = "black --check --diff hermeto tests noxfile.py"
     session.run(*cmd.split(), *session.posargs, silent=True)
 
 
 @nox.session()
 def flake8(session: Session) -> None:
-    """Run flake8 on cachi2 and tests directories and noxfile.py."""
+    """Run flake8."""
```

Comment:
These docstrings are not really helpful and could be dropped entirely, should we just turn off the corresponding warning for flake8 for this module? (out of scope for this change, this non-actionable)

Reply:
I'd rather satisfy flake with unhelpful doc strings since the code is self explanatory rather than disabling rules. It's cheaper IMO and more transparent.